<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeaturesWindow</class>
 <widget class="QDialog" name="FeaturesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(255, 178, 25)
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>201</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:black
}</string>
   </property>
   <property name="text">
    <string>Features Window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WithdrawBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:white
}</string>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DepositBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:white
}</string>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:white
}</string>
   </property>
   <property name="text">
    <string>Transfer</string>
   </property>
  </widget>
  <widget class="QLabel" name="BalanceLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Balance:   10,212.00 THB</string>
   </property>
  </widget>
  <widget class="QLabel" name="AccNumLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Account Number: 1110-01023-4122</string>
   </property>
  </widget>
  <widget class="QLabel" name="UsernamLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>Username:  Thanaphat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeBtn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>453</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	background-color:white
}</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
